package.json

```diff
@@ -48,31 +48,46 @@
   "scripts": {
     "docs": "repo-tools exec -- jsdoc -c .jsdoc.js",
     "generate-scaffolding": "repo-tools generate all --except lib_readme",
-    "lint": "repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/",
+    "lint": "cd samples/ && npm link ../ && repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/",
```

Comment:
I am not sure why this is necessary. If `repo-tools` doesn't do what you want it to do, perhaps we need to add a new feature to it?

Reply:
lint gave error on sample before
24:32  error  "@google-cloud/logging-winston" is not found  node/no-missing-require
I am actually going to change to.
"lint": "cd samples/ && npm link ../ && cd .. && repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/",